content/en/docs/workflow.md

```diff
@@ -1,49 +1,41 @@
 ---
 title: "Workflow"
 excerpt: ""
-weight: 13
+weight: 2
 ---
 
-If you know the basic commands, have gone through [Getting Started](doc:getting-started), [Basic Usage](doc:basic-usage), and have a brief understanding of [Releases](doc:releases), the next step is possibly to figure out a workflow for your project and your team.
+If you know the basic commands, have gone through [Getting Started](/docs/getting-started), [Basic Usage](/docs/basic-usage), and have a brief understanding of [Releases](/docs/releases), the next step is possibly to figure out a workflow for your project and your team.
 
 This section is a work in progress of various recommended steps and possible default configurations for various tasks in order to have a good experience with the Erlang toolchain.
 
 ## Pick the Right Type of Project
 
 |Type of Project|Recommended Template|Comments|
 |----|----|----|
-|A short script or util|escript|The user you distribute this to will need to have Erlang installed.
-
-Dependencies in C are not trivially inclduded nor redistributed|
-|A full, self-contained, executable system|release or umbrella|This is the recommended production deploy for Erlang systems.
-
-See the [Releases](doc:releases) section for more details on releases|
-|A library to be used in other systems|lib or app|use 'lib' for stateless libraries that contain modules, and 'app' for stateful libraries with a supervision tree|
-|A collection of multiple libraries|umbrella|This is the one form of project supported where multiple top-level apps are used.
-
-These projects cannot be used as dependencies.
-
-For projects usable as dependencies, see [Vendoring Plugins](https://www.rebar3.org/v3/docs/using-available-plugins#section-vendoring-dependencies)|
+|A short script or util|escript|The user you distribute this to will need to have Erlang installed.<br><br>Dependencies in C are not trivially inclduded nor redistributed|
+|A full, self-contained, executable system|release or umbrella|This is the recommended production deploy for Erlang systems.<br><br>See the [Releases](/docs/releases) section for more details on releases|
+|A library to be used in other systems|lib or app|use `lib` for stateless libraries that contain modules, and `app` for stateful libraries with a supervision tree|
+|A collection of multiple libraries|umbrella|This is the one form of project supported where multiple top-level apps are used.<br><br>These projects cannot be used as dependencies.<br>For projects usable as dependencies, see [Vendoring Plugins](https://www.rebar3.org/v3/docs/using-available-plugins#section-vendoring-dependencies)|
```

Comment:
Why does this link to the vendoing project to talk about "projects usable as dependencies"?

Reply:
This is in the context of an umbrella applications when that was written on the site.. I should probably change it and mention git_subdir deps instead now.